src/vivarium_cluster_tools/psimulate/redis_dbs/registry.py

```diff
@@ -136,21 +135,23 @@ def _update_status(self) -> None:
                 q_workers = len(rq.Worker.all(queue=self._queue)) if self._queue else 0
 
                 self._status["pending"] = len(pending_ids)
-                self._status["running"] = len(running_ids) + len(finished_ids)
+                self._status["running"] = len(running_ids) + len(successful_ids)
                 self._status["failed"] = len(failed_ids)
-                self._status["finished"] = self._status["finished"]
+                self._status["successful"] = self._status["successful"]
                 self._status["total"] = (
                     len(pending_ids)
                     + len(running_ids)
                     + len(failed_ids)
-                    + self._status["finished"]
+                    + self._status["successful"]
                 )
                 self._status["workers"] = q_workers
-                self._status["done"] = 100 * self._status["finished"] / self._status["total"]
+                self._status["done"] = (
```

Comment:
so "done" for this context is basically finished + successful?

Reply:
Yeah, I don't think that the "done" percentage is including failed jobs